tests/functional/d/deprecated/deprecated_module_py39.rc

```diff
@@ -0,0 +1,5 @@
+[testoptions]
+min_pyver=3.9
+
+[Messages Control]
+enable=deprecated-module
```

Comment:
Isn't it enabled by default?

Reply:
Thanks, this was a left-over artifact from investigating unexpected test results.